queue/src/fiskaltrust.Middleware.Localization.QueueDE/JournalProcessorDE.cs

```diff
@@ -151,7 +155,10 @@ private async IAsyncEnumerable<JournalResponse> ProcessTarExportFromTSEAsync(Jou
         {
             var exportSession = await _deSSCDProvider.Instance.StartExportSessionAsync(new StartExportSessionRequest()).ConfigureAwait(false);
             var sha256CheckSum = "";
-            using (var memoryStream = new MemoryStream())
+
+            byte[] chunk;
+            var response = new JournalResponse();
+            using (var stream = new FileStream($"{exportSession.TokenId}.temp", FileMode.Create, FileAccess.ReadWrite))
```

Comment:
I think it's nicer to put this file in the temp directory 🤔
```suggestion
            var tempFilePath = Path.Combine(Path.GetTempPath(), "fiskaltrust", $"{exportSession.TokenId}.temp");
            using (var stream = new FileStream(tempFilePath, FileMode.Create, FileAccess.ReadWrite))
```